### PR TITLE
*: update proto

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut

### CODE REVIEW

File: proto/pd_jobpb.proto [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    // When pd server gets the first job and before handling, it must first change
+    // the job's status to running. If the job is running, we should check whether
+    // the job is already finished in raft server.
+    Running = 1;
+
+    Finished = 2;
+    Canceled = 3;
+}
+
+// For every conf change, split operation, we will add this into job list,
```
Comment (ngaut) [inline proto/pd_jobpb.proto]: s/every/each

Comment (ngaut) [inline proto/pd_jobpb.proto]: s/list/queue

File: proto/raft_cmdpb.proto [hunk 2]
```diff
@@ -128,6 +129,24 @@ message RegionLeaderResponse {
     optional uint64 current_term 	= 2;
 }
 
+// For get more information of the region.
```
Comment (ngaut) [inline proto/raft_cmdpb.proto]: s/get/getting

**Comment by siddontang** [comment]
PTAL

**Comment by ngaut** [comment]
LGTM

**Pull request merged**